build/docker/proxy/Dockerfile

```diff
@@ -3,6 +3,7 @@ FROM nginx:latest
 COPY assets/certificate/certificate.key /certificate/certificate.key
 COPY assets/certificate/certificate.pem /certificate/certificate.pem
 COPY assets/etc/nginx/nginx.conf /etc/nginx/nginx.conf
+COPY assets/var/www/capacity-status-api/static/ /var/www/capacity-status-api/static
```

Comment:
Can we change the directory of the static files to `assets/application`? This is for consistency as we do that in the other type of containers.